whisper/transcribe.py

```diff
@@ -216,6 +219,13 @@ def decode_with_fallback(segment: torch.Tensor) -> DecodingResult:
                 and decode_result.no_speech_prob > no_speech_threshold
             ):
                 needs_fallback = False  # silence
+            if (
+            compression_ratio_hallucination_threshold is not None
```

Comment:
The condition should be indented one more level to match the style of the multi-line if conditions around it)

Reply:
Fair enough, fixed